learn4haskell.cabal

```
@@ -22,7 +22,7 @@ source-repository head
   location:            https://github.com/kowainik/learn4haskell.git
 
 common common-options
-  build-depends:       base ^>= 4.14.0.0
+  build-depends:       base ^>= 4.12.0.0
```

We wrote the infrastructure so that it works with the latest version of GHC – 8.10. So in order to work with that you need no allow `base-4.14.0.0` as well.
```suggestion
  build-depends:       base >= 4.12.0.0 && < 4.15
```

Also, you can enable GitHub Actions CI by enabling it here https://github.com/asjad02/learn4haskell/actions